<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KampfWindow</class>
 <widget class="QDialog" name="KampfWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kampf</string>
  </property>
  <widget class="QLabel" name="label2HPmon">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>475</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>680</y>
     <width>491</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2HP">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>475</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HP:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1Att1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text1Mon1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1Att2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelKampfName2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>230</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text1Mon2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QProgressBar" name="lifeBar1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>181</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 194, 17);
selection-background-color: rgb(38, 162, 105);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelBild2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text2Mon3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text1Mon3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Att1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Kampf starten</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelBild1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label1HPmon">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>475</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text2Mon2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text2Mon1">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Att2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelKampfName1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>230</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1HP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>475</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HP:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="lifeBar2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>181</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 194, 17);
selection-background-color: rgb(38, 162, 105);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Auswahl2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>auswählen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Auswahl3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>auswählen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1Auswahl2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>auswählen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1Auswahl3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>auswählen</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelKonfetti">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="lvlBar1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>181</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
selection-background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="lvlBar2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>181</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
selection-background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelLevel1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelLevel2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>500</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btnStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn1Auswahl2</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btn1Auswahl2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn1Auswahl3</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btn1Auswahl3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn2Auswahl2</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btn2Auswahl2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn2Auswahl3</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btn2Auswahl3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn1Att1</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btn1Att1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn1Att2</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btn1Att2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn2Att2</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btn2Att2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>753</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn2Att1</sender>
   <signal>clicked()</signal>
   <receiver>KampfWindow</receiver>
   <slot>btn2Att1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnStart()</slot>
  <slot>btn1Att1()</slot>
  <slot>btn1Att2()</slot>
  <slot>btn2Att1()</slot>
  <slot>btn2Att2()</slot>
  <slot>btn1Auswahl2()</slot>
  <slot>btn1Auswahl3()</slot>
  <slot>btn2Auswahl2()</slot>
  <slot>btn2Auswahl3()</slot>
 </slots>
</ui>
